tests/unit/test_charm.py

```diff
@@ -186,6 +186,7 @@ def test_ready(self):
                             self.harness.model.get_binding("peer").network.ingress_address
                         ),
                         "NUM_HISTORY_SHARDS": 1,
+                        "SQL_TLS_ENABLED": False,
```

Comment:
Suggestion: It might be worth having a unit test where this value is `True` to validate the logic.